src/forcing.jl

```diff
@@ -0,0 +1,42 @@
+"""
+    SimpleForcing([location=(Cell, Cell, Cell),] forcing_function::Function)
+
+Construct forcing for a field at `location` from `forcing_function(x, y, z, t)`, 
+where `x, y, z` is position and `t` is time.
+"""
+struct SimpleForcing{Lx, Ly, Lz, F}
+    func :: F
+end
+
+"""
+    SimpleForcing([location=(Cell, Cell, Cell),] forcing::SimpleForcing)
+
+Construct forcing for a field at `location` using `forcing.func`.
+"""
```

Comment:
I might add an example to the docstring like the one from the PR comment
```julia
T_forcing = SimpleForcing((x, y, z, t) -> exp(z) * cos(t))
```
or for people who don't know about anonymous functions
```julia
FT(x, y, z, t) = exp(z) * cos(t)
T_forcing = SimpleForcing(FT)
```